i18n/de_DE.csv

```diff
@@ -0,0 +1,3 @@
+"We have noticed that you are visiting this website from ""%1"".","Wir haben festgestellt, dass Sie diese Webseite aus ""%1"" besuchen"
+"If we have set the country incorrectly, please change the country using our country list (at the top right of the screen).","Sollten wir das Land falsch festgelegt haben, dann ändern Sie das Land über unsere Länderliste (am oberen rechten Bildschirmrand)."
+"Thank you very much and enjoy your visit in the SATA-Shop.","Vielen Dank und viel Freude bei Ihrem Einkauf im SATA-Shop"
```

Comment:
Nope. Customer name shouldn't be in a puplic package!

Reply:
fixed